src/workspaces.c

```diff
@@ -260,7 +292,17 @@ _osd_show(struct server *server)
 void
 workspaces_init(struct server *server)
 {
-	wl_list_init(&server->workspaces);
+	server->workspaces.cosmic_manager = wlr_cosmic_workspace_manager_create(
```

Comment:
I'm sure it's right, but why do we pass `CW_CAP_WS_ACTIVATE` in `wlr_cosmic_workspace_manager_create()`